plugins/MauticCitrixBundle/Form/Type/CitrixActionType.php

```diff
@@ -119,14 +121,15 @@ function ($p) {
         ) {
             $builder->add(
                 'firstname',
-                'choice',
+                ChoiceType::class,
                 [
-                    'choices'    => $choices,
-                    'expanded'   => false,
-                    'label_attr' => ['class' => 'control-label'],
-                    'multiple'   => false,
-                    'label'      => 'plugin.citrix.first_name',
-                    'attr'       => [
+                    'choices'           => array_flip($choices),
```

Comment:
You can flip $choices in foreach on line 82:
`$choices[$f['label']] = $f['id'];`

Reply:
Fixed 6febf86ac7d6df59dcc78faa327c5803a1229bcf